soepy/solve/solve_auxiliary.py

```diff
@@ -290,11 +291,37 @@ def construct_covariates(states, model_spec):
         + model_spec.partner_cf_educ * states[:, 1]
     )
 
+    # Male wages
     # Final input of male wages / partner income is calculated on a weekly
     # basis. Underlying assumption that all men work full time.
     male_wages = np.where(states[:, 7] == 1, np.exp(log_wages) * HOURS[2], 0)
 
-    covariates = np.column_stack((bins, male_wages))
+    # Equivalence scale
```

Comment:
Please refactor into a function and write some hard-coded test cases .. appears to be just one all the time. Is this for regression tests?